cypress/integration/events_test.spec.ts

```diff
@@ -1,107 +1,393 @@
 import LoginPage from "../support/pages/LoginPage";
 import SidebarPage from "../support/pages/admin_console/SidebarPage";
-import UserEventsTab from "../support/pages/admin_console/manage/events/UserEventsTab";
-import AdminEventsTab from "../support/pages/admin_console/manage/events/AdminEventsTab";
+import UserEventsTab, {
+  UserEventSearchData,
+} from "../support/pages/admin_console/manage/events/tabs/UserEventsTab";
+import AdminEventsTab from "../support/pages/admin_console/manage/events/tabs/AdminEventsTab";
 import RealmSettingsPage from "../support/pages/admin_console/manage/realm_settings/RealmSettingsPage";
 import Masthead from "../support/pages/admin_console/Masthead";
 import { keycloakBefore } from "../support/util/keycloak_hooks";
+import EventsPage, {
+  EventsTab,
+} from "../support/pages/admin_console/manage/events/EventsPage";
+import ListingPage from "../support/pages/admin_console/ListingPage";
+import adminClient from "../support/util/AdminClient";
 
 const loginPage = new LoginPage();
 const sidebarPage = new SidebarPage();
 const userEventsTab = new UserEventsTab();
+const eventsPage = new EventsPage();
 const adminEventsTab = new AdminEventsTab();
 const realmSettingsPage = new RealmSettingsPage();
 const masthead = new Masthead();
+const listingPage = new ListingPage();
+
+const dateFrom = new Date();
+dateFrom.setDate(dateFrom.getDate() - 100);
+const dateFromFormatted = `${dateFrom.getFullYear()}-${dateFrom.getMonth()}-${dateFrom.getDay()}`;
+const dateTo = new Date();
+dateTo.setDate(dateTo.getDate() + 100);
+const dateToFormatted = `${dateTo.getFullYear()}-${dateTo.getMonth()}-${dateTo.getDay()}`;
 
 describe("Events tests", () => {
-  describe.skip("Search user events", () => {
+  const eventsTestUser = {
+    eventsTestUserId: "",
+    userRepresentation: {
+      username: "events-test",
+      enabled: true,
+      credentials: [{ value: "events-test" }],
+    },
+  };
+  const eventsTestUserClientId = "admin-cli";
+
+  before(async () => {
+    const result = await adminClient.createUser(
+      eventsTestUser.userRepresentation
+    );
+    eventsTestUser.eventsTestUserId = result.id;
+  });
+
+  after(async () => {
+    await adminClient.deleteUser(eventsTestUser.userRepresentation.username);
+  });
```

Comment:
No need to mark this as `async` if a Promise is returned.
```suggestion
  after(() => adminClient.deleteUser(eventsTestUser.userRepresentation.username));
```